README.md

```diff
@@ -0,0 +1,10 @@
+# GNAT2GOTO
+An [Ada](https://en.wikipedia.org/wiki/Ada_(programming_language)) frontend for
+[CBMC](https://github.com/diffblue/cbmc). Named thus because it is implemented
+as a backend for the [GNAT Ada Compiler](https://www.gnu.org/software/gnat/)
+(see also: [AdaCore](https://www.adacore.com/)), that produces a GOTO program
+(the internal program representation of CBMC) in JSON format.
+
+For instructions on building the project, see [gnat2goto/BUILDING.md](gnat2goto/BUILDING.md)
+
+For an overview of the project, see the [Wiki](https://github.com/diffblue/gnat2goto/wiki) (work in progress)
```

Comment:
Drop the 'work in progress' - it's *always* going to be a work in progress :-)

Reply:
Ah, misplaced optimism on my part